runtime/zeitgeist/src/integration_tests/xcm/tests/transfers.rs

```diff
@@ -351,6 +360,13 @@ fn transfer_eth_sibling_to_zeitgeist() {
             Tokens::free_balance(ETH_ID, &ALICE),
             zeitgeist_alice_initial_balance + expected_adjusted,
         );
+
+        // Verify that fees (of foreign currency) have been put into treasury
+        assert_eq!(
+            Tokens::free_balance(ETH_ID, &ZeitgeistTreasuryAccount::get()),
+            // Align decimal fractional places
+            treasury_initial_balance + (eth_fee() * ztg(1)) / eth(1)
```

Comment:
Could have also used `adjusted_balance` here.

Reply:
Not even a year passed after I wrote that function and I already forgot :sweat_smile: 

https://github.com/zeitgeistpm/zeitgeist/pull/1225/commits/b019bb87aa31d38c699b82359bf24103cd49c8de